compiler/next/test/parsing/testParseStringBytesLiterals.cpp

```diff
@@ -64,7 +64,9 @@ static void testStringLiteral(Parser* parser,
   auto strLit = initExpr->toStringLiteral();
   assert(strLit);
   assert(strLit->quoteStyle() == expectQuoteStyle);
-  assert(strLit->str().toString() == expectValue);
```

Comment:
`assert(strLit->str() == expectValue);` or if that doesn't work `assert(strLit->str().str() == expectValue);`